File: FRONTEND/FRONTEND/src/app/service.ts
import { Employee } from "./employee"
import { Order } from "./order"

export class Service {
    id: string = ""
    name: string= ""
    description: string = ""
    orders?: Order[]
    employees?: Employee[]
    editMode: boolean = false

      constructor(init?: Partial<Service>) {
    Object.assign(this, init)
  }
}

File: BACKEND/Entities/Models/Service.cs
using Entities.Helpers;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace Entities.Models
{
    public class Service : IIdEntity
    {
        [Key]
        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        public string Id { get; set; }
        [StringLength(100)]
        public string Name { get; set; }
        [StringLength(500)]
        public string Description { get; set; }
        public virtual ICollection<Order> Orders { get; set; }

        public virtual ICollection<Employee>? Employees { get; set; }

        public Service(string name, string description)
        {
            Id = Guid.NewGuid().ToString();
            Name = name;
            Description = description;
        }

        public Service()
        {
        }
    }
}
